conda/recipes/cudf/meta.yaml

```diff
@@ -46,6 +46,7 @@ requirements:
     - cupy >=9.5.0,<11.0.0a0
     - numba >=0.54
     - numpy
+    - s3fs >=2022.3.0
```

Comment:
I presume this means that from this release cudf will start shipping with s3fs. We should also consider if we want to ship with other similar remote filesystems like gcsfs. 

Note: The default behavior for pandas/dask here is to raise an error stating that the optional dependency for s3fs needs to be installed.